Approaching a transformation with metafacture.md

```diff
@@ -0,0 +1,15 @@
+Every approach transform metadata with metafacture is quite similiar:
+
+- You need to know what you want:
+	e.g. Transform data from Marc21 from a certain folder to some kind of JSON Data.
+- You have to identify the commands that you need.
+- Combine the commands without the transformation module and test if the workflow goes trough.
```

Comment:
Combine the commands without the transformation module and test if the workflow goes through.